extensions/amp-script/amp-script.md

```diff
@@ -77,63 +109,103 @@ You can also include your JavaScript inline, in a `script` tag. You must:
+  h1.textContent = 'Hello World!';
+  document.body.appendChild(h1);
+});
 ```
 
-Under the hood, `amp-script` uses [@ampproject/worker-dom](https://github.com/ampproject/worker-dom/). For design details, see the ["Intent to Implement" issue](https://github.com/ampproject/amphtml/issues/13471).
+Supported DOM APIs include:
 
-## State manipulation
+* Element getters like `getElementByName()`, `getElementsByClassName()`, `getElementsByTagName()`, `childNodes()`, `parentNode()`, and `lastChild()`
+* Mutators like `createTextNode()`, `appendChild()`, `insertBefore()`, `removeChild()`, and `replaceChild()`
+* Methods involving events like `addEventListener()`, `removeEventListener()`, and `createEvent()`
+* Property and attribute getters like `getAttribute()`, `hasAttribute()`
+* Event properties like `Event.target`, `Event.type`, and `Event.bubbles`
+* Element properties like `attributes`, `id`, `outerHTML`, `textContent`, `value`, `classList`, and `className`
+* And many more.
 
-`amp-script` supports getting and setting [`amp-state`](https://amp.dev/documentation/components/amp-bind/#initializing-state-with-amp-state) JSON via JavaScript.
+`querySelector()` is supported for simple selectors - element, ID, class, and attribute. So, `document.querySelector('.class')` will work, but `document.querySelector('.class1 .class2')` will not. [See the code](https://github.com/ampproject/worker-dom/blob/main/src/worker-thread/dom/Element.ts#L159) for details.
 
-This enables advanced interactions between `amp-script` and other AMP elements on the page via `amp-bind` [bindings](https://amp.dev/documentation/components/amp-bind/#bindings). Invoking `AMP.setState()` from `amp-script` may cause mutations to the DOM as long as it was triggered by user gesture, otherwise it will only implicitly set state (similar to [`amp-state` initialization](https://amp.dev/documentation/examples/components/amp-bind/?referrer=ampbyexample.com#initializing-state)).
+`amp-script` supports common Web APIs like `Fetch`, `Websockets`, `localStorage`, `sessionStorage`, and `Canvas`. Presently, the `History` API is not implemented, and neither are cookies.
+
+`amp-script` does not support the entire DOM API or Web API, as this would make `amp-script`'s own JavaScript too large and slow. If there's an API you'd like to see supported, please [file an issue](https://github.com/ampproject/amphtml/issues/new) or [suggest and contribute the change yourself](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md). For a complete list of supported DOM APIs, see the [API compatibility table](https://github.com/ampproject/worker-dom/blob/master/web_compat_table.md).
 
 [tip type="default"]
-`AMP.setState()` requires the [`amp-bind`](https://amp.dev/documentation/components/amp-bind) extension script to be included in the document head.
+For a set of samples showing `amp-script` in use, [see here](/documentation/examples/components/amp-script/).
```

Comment:
I don't think this link will work - @matthiasrohmer do you know?

Reply:
It works in my dev environment, at least. Of course, arguably it's redundant because there's a small link to this at the top of the page. I  just thought it would be a good time for readers to see some more examples.

Reply:
I try and do local links for the `amphtml` repo... but those might end up sending readers to GitHub. I'll add it to the amp.dev agenda for tomorrow.

Reply:
Made absolute as per our conversation.

Reply:
Thanks so much Ben!